docker-compose.bhutan.yaml

```
@@ -69,6 +69,9 @@ services:
       TWILIO_AUTH_TOKEN:
       TWILIO_VERIFY_SERVICE_SID:
       TWILIO_MESSAGING_SERVICE_SID:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME:
```

Rather use shell interpolation here:
```suggestion
      WALLET_ADDRESS_DB_CONNECTION_STRING: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING}
      WALLET_ADDRESS_DB_DATABASE_NAME: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME}
      WALLET_ADDRESS_DB_COLLECTION_NAME: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME}
```
This way all the containers use the same environment variable but they're value will be derived from a separate entry in the `.env` file.  You generally want your configuration to be as uniform as possible.